slash/leave.js

```diff
@@ -1,9 +1,9 @@
-const { Permissions } = require("discord.js");
-
 exports.run = async (client, interaction) => { // eslint-disable-line no-unused-vars
```

Comment:
You could prefix the unused vars with `_` instead of using `eslint-disable-line`

Reply:
Not something I wrote and not really relevant to this pr. Also I feel like this might have had a reason for this, as I'm like 90% sure it doesn't yell at unused variables before.